RELEASE_NOTES.md

```diff
@@ -4,6 +4,9 @@ Changes:
 
   * Added configuration for library aliquot alias generator and validator
   * Added V2 naming scheme
```

Comment:
Might be worth changing this to `v2` to match the case expected in `miso.properties`

Reply:
Changed

Reply:
Looks good to me